codice/ddf
# DDF-2094 Add support for decrypting encrypted passwords in sources

#### What does this PR do?
The following sources now support decrypting encrypted passwords:
catalog-opensearch-source
spatial-csw-source
spatial-wfs-v2_0_0-source
spatial-wfs-v1_0_0-source
catalog-registry-api-impl
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@ryeats @coyotesqrl @tbatie @jckilmer @bcwaters @kcwire 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@shaundmorris
@stustison 
#### How should this be tested?
Enter an unencrypted password in a password field of a source configuration. Check the logs and verify it says a clear text password has been passed in. If possible, encrypt the password and pass it in a source configuration and verify the source works.
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-2094
#### Screenshots (if appropriate)
#### Checklist:
